backend-api/template.yaml

```diff
@@ -327,7 +335,16 @@ Resources:
                   - kms:Decrypt
                   - kms:GenerateDataKey
                 Resource: !GetAtt TxMAKMSEncryptionKey.Arn
-      RoleName: !Sub ${AWS::StackName}-token-lambda
+        - PolicyName: ENIPolicy
```

Comment:
This could have been done another way using `ManagedPolicyArns`, which would reduce the size of the template but maybe adds more permissions than needed. Any opinions on which option would work best here?